src/content/docs/guides/configure-biome.mdx

```diff
@@ -187,33 +178,36 @@ Some shells don't support the recursive glob `**`.
 :::
 
 The Biome configuration file can be used to refine which files are processed.
-You can explicitly list the files to be processed using `include` and the files not to be processed using `ignore`.
-`include` and `ignore` accepts globs patterns such as `src/**/*.js`.
-See the [related section](#glob-syntax-explained) for which glob syntaxes are supported.
-`include` is always applied first before applying `ignore`.
-This allows you to include some files and to ignore some of the file you included.
-
-:::note
-`include` and `ignore` have a slightly different semantics.
-`include` doesn't prevent Biome of traversing a folder.
-This means that if you want to prevent Biome from traversing a folder, you have to add the folder to `ignore`.
+You can explicitly list the files to be processed using
+[the `files.includes` field](/reference/configuration). `files.includes` accepts
+[glob patterns](/reference/configuration#glob-syntax-reference) such as
+`src/**/*.js`. Negated patterns starting with `!` can be used to exclude files.
+
+Paths and globs inside Biome's configuration file are always resolved relative
+to the folder the configuration file is in.
+
+`files.includes` applies to all of Biome's tools, meaning the files specified
+here are processed by the linter, the formatter and the assist, unless specified
+otherwise. For the individual tools, you can further refine the matching files
+using `<tool>.includes`.
+
+:::important
```

Comment:
This isn't part of [starlight](https://starlight.astro.build/guides/authoring-content/#asides). Did you want to use `:::note` or `:::caution`?

Reply:
Thanks, looks like I looked at a different markdown flavour reference 😅